src/applications/resources-and-support/hooks/useArticleData.js

```diff
@@ -1,6 +1,12 @@
 import { useEffect, useState } from 'react';
 import * as Sentry from '@sentry/browser';
 
+// We use CMS demo environments to publish articles in bulk.
+// This means that at this time, we can't use the environment.BASE_URL,
+// because the CMS demo env uses domains not defined there, and we need
+// this search app to align with the demo.
+const baseUrl = document.location.origin;
```

Comment:
Thanks for this comment! 🔥

Reply:
It's still going to confuse me in a few weeks from now lol